Cargo.lock

```diff
@@ -17461,8 +17461,26 @@ dependencies = [
 [[package]]
 name = "tonic"
 version = "0.13.0"
+source = "git+https://github.com/hyperium/tonic.git?rev=fee9c130c86a365b039f3e70a72a94ea28a9aaa9#fee9c130c86a365b039f3e70a72a94ea28a9aaa9"
+dependencies = [
+ "base64 0.22.1",
+ "bytes",
+ "http 1.1.0",
+ "http-body 1.0.1",
+ "http-body-util",
+ "percent-encoding",
+ "pin-project",
+ "tokio-stream",
+ "tower-layer",
+ "tower-service",
+ "tracing",
+]
```

Comment:
This looks odd, can we check that tonic 0.13.0 isn't used anywhere anymore and it should be removed